### ISSUE TITLE
Spotter is visually very broken with the Dark Theme

### ISSUE THREAD

**Issue by zilti** [guillep/pharo-core#11167]
When activating the Dark Theme in Pharo 10, the Spotter keeps a white background like in the Light Theme, but the font colours, the scroll bar, and the mouseovers change to the Dark Theme.

**Comment by sonibla** [comment]
I found out how to reproduce the bug
In a Pharo 10 image, open a playground, then settings, switch to dark theme, open a spotter

![Pharo_A9OcQ7CGgK](https://user-images.githubusercontent.com/49032546/214523805-eb1fad4d-70aa-43ae-8d47-70767c17b4a0.gif)

Spotter has expected colors if you don't open any playground before changing theme... strange but at least the issue seems to be fixed in Pharo 11

Reproduced the bug on
Pharo 10.0.0
Build information: Pharo-10.0.0+build.536.sha.2314c3f457171dcde477ff6575b578835f1fd519 (64 Bit)
Windows 10 & Linux

**Comment by estebanlm** [comment]
this is a refresh theme problem, not a problem with spotter itself :)

**Comment by estebanlm** [comment]
however, what I do realise is that spotter theming is broken since the introduction of color palettes. 
@jordanmontt any idea?

**Comment by jordanmontt** [comment]
The color paletteit was only introduced for Pharo 11, not Pharo 10. So I don't think that the palette is the cause.

Indeed for Pharo 11 there is a color problem for the Spotter, but that is another issue

**Comment by estebanlm** [comment]
yeahm, I highjaked this issue (which in fact is fixed)
